cpu/esp8266/doc.txt

```diff
@@ -339,24 +340,35 @@ Option | Values | Default | Description
 -------|--------|---------|------------
 ENABLE_GDB | 0, 1 | 0 | Enable compilation with debug information for debugging with QEMU (```QEMU=1```), see section [QEMU Mode and GDB](#esp8266_qemu_mode_and_gdb)
 FLASH_MODE | dout, dio, qout, qio | dout | Set the flash mode, please take care with your module, see section [Flash Modes](#esp8266_flash_modes)
-NETDEV_DEFAULT | module name | mrf24j40 | Set the module that is used as default network device, see section [Network Devices](#esp8266_network_devices)
-PORT | /dev/ttyUSBx | /dev/ttyUSB0 | Set the USB port for flashing the firmware
+PORT | /dev/ttyUSBx | /dev/* | Set the USB port for flashing the firmware
 QEMU | 0, 1 | 0 | Generate an image for QEMU, see section [QEMU Mode and GDB](#esp8266_qemu_mode_and_gdb).
 USE_SDK | 0, 1 | 0 | Compile the SDK version (```USE_SDK=1```), see section [SDK Task Handling](#esp8266_sdk_task_handling)
 
 </center><br>
 
-Optional features of ESP8266 can be enable by ```USEMODULE``` definitions in the makefile of the application. These are:
+Optional features of ESP8266 can be enabled by ```USEMODULE``` definitions in the makefile of the application. These are:
 
 <center>
 
 Module | Description
 -------|------------
-esp_spiffs | Enables the SPIFFS file system, see section [SPIFFS Device](#esp8266_spiffs_device)
-esp_sw_timer | Enables software timer implementation, implies the setting ```USE_SDK=1```, see section [Timers](#esp8266_timers)
+[esp_gdb](#esp8266_qemu_mode_and_gdb) | Enable the compilation with debug information, which is equivalent to using ```ENABLE_GDB=1```
+[esp_sdk](#esp8266_sdk_task_handling) | Enable the SDK version, which is equivalent to using ```USE_SDK=1```
+[esp_spiffs](#esp8266_spiffs_device) | Enable the SPIFFS drive in on-board flash memory
+[esp_sw_timer](#esp8266_timers) | Enable software timer implementation, implies the setting ```USE_SDK=1``` (module ```esp_sdk```)
 
 </center><br>
 
+For example, to activate the a SPIFFS drive in on-board flash memory, the makefile of application has simply to add the ```esp_spiffs``` module to ```USEMODULE``` make variable:
```

Comment:
Extra `a` copy pasted from the README.md. Just a thought: can we just keep one of these files ? (doc.txt)

Reply:
`README.md` and `doc.txt` are identical. I use `README.md` to write the documentation to have immediate feedback in markdown viewer. It is quite tedious to write complex documentation for `doxygen` and execute `make doc` to see the results.

Using `doc.txt` is the new approach for board documentation. So, if only one file should be kept, it should be `doc.txt` :worried:

Reply:
I kept `README.md` for the moment so that you can track requested changes. I will remove it before squashing.